### PR TITLE
Cleanup fetching and count

### PR DESCRIPTION
**Pull request by kostiakoval**
Cleanup proposal.If you agree can cleanup further. 
Counting and fetching should go to some base method.

### CODE REVIEW

File: Tests/Tests/Tests.m [hunk 1]
```diff
@@ -46,6 +46,50 @@ - (NSArray *)objectsFromJSON:(NSString *)fileName {
...
+    NSError *error = nil;
+    NSFetchRequest *fetch = [NSFetchRequest fetchRequestWithEntityName:entity];
+    fetch.predicate = predicate;
+    NSInteger count = [context countForFetchRequest:fetch error:&error];
+    if (error) NSLog(@"countError: %@", [error description]);
+
+    return count;
+}
+
+- (void)test:(NSString *)predicateFormat, ... {
```
Comment (3lvis) [inline Tests/Tests/Tests.m]: What's this for?

**Comment by 3lvis** [comment]
Looks good @kostiakoval! What's left to remove the WIP badge?

File: Tests/Tests/Tests.m (hunk 1 above)
Comment (kostiakoval) [inline Tests/Tests/Tests.m]: Ups, forgot to remove it :)

**Comment by 3lvis** [comment]
Ideally the helper methods should me moved to a category of `NSManagedObject` or NSEntityDescription

**Pull request merged**